docs/recent_activity/recent_activity.md

```diff
@@ -1,70 +1,54 @@
 ---
-description: Monitor recent activity logged actions.
+description: Monitor recently logged user actions.
+edition: experience
 ---
 
-# Recent activity log [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+# Review recent user activity
 
-## Activity list
+[[= product_name =]] logs various operations on the repository and in the application.
+If you have administrator permissions and your [user role](permission_system.md) has the **Activity Log / Read** permission, you can review the most recent activity log in the Back Office, the Admin tab.
 
-In the Back Office, **Admin** -> **Activity list**, you can see a list of recent activity of all or selected users.
+Depending on the system configuration, activity logs may also be shown:
+
+- on the Dashboard, the [Recent activity block](dashboard_block_reference.md#recent-activity-block)
+- within the [user profile](../getting_started/get_started.md#view-and-edit-user-profile)
 
 !!! note
 
-    To see the **Admin** menu, you need the **Setup / Administrate** [permission](permission_system.md).
-    To see the Activity list, you need the **Activity Log / Read** permission (this permission can be limited to allow seeing your own activity only).
+    By default, log entries are kept for 30 days.
+    This time can be modified through configuration.
+    For more information, see [developer documentation]([[= developer_doc =]]/administration/recent_activity/recent_activity/#configuration-and-cronjob).
+
+## View user activity in Back Office
 
-By default, the following actions are displayed:
+In the Back Office, go to **Admin** -> **Activity list**, to see recent activity of all or selected users.
+
+The following actions are displayed:
 
 - [Content](content_items.md) create, create draft, publish, update, trash, recover, delete, delete translation, hide, and reveal
 - [Location](manage_locations_urls.md#content-locations) create, delete, hide, reveal, update, move, swap, and subtree copy
 - [Product](products.md) create, update, and delete
 - [Product variant](work_with_product_variants.md) create, update, and delete
 - [Site](work_with_sites.md) create, update, and delete
```

Comment:
It should be put into the table for better reading experience.

Reply:
@juskora  Or not listed at all.

```
All actions on the following items are logged:

- [Content](content_items.md)
- [Location](manage_locations_urls.md#content-locations)
- [Product](products.md)
- [Product variant](work_with_product_variants.md)
- [Site](work_with_sites.md)
```

Reply:
I was trying to be exhaustive by reminding all possible actions on each item.
From my POV, a table would convey a wrong message about what is supported and what's not. But maybe I think that because I don't see the same table.

|Type|publish|
|-----|-------|
|Content| ✔️ |
|Location|  |

This kind of table say that Location/publish is not logged. It doesn't say that there is nothing to log as Location/publish doesn't exist.